js/build_map.js

```diff
@@ -137,12 +139,11 @@ var categoryButtons = d3.select('#button-container')
   .on('click', function(d){
     prevCategory = activeCategory;
     activeCategory = d.toLowerCase(); // put this here so it only changes on click
```

Comment:
looks like the above two lines may now be redundant with map_utils.js lines 159-160? i think you could take out these two here, then pass `d.toLowerCase()` to updateCategory (or do the lowercasing within updateCategory, whichever)